docs/installation.md

```diff
@@ -5,14 +5,14 @@
 Add repository setting to `/etc/yum.repos.d`.
 
 ```bash
-$ sudo vim /etc/yum.repos.d/trivy.repo
-[trivy]
-name=Trivy repository
-baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$releasever/$basearch/
-gpgcheck=0
-enabled=1
-$ sudo yum -y update
-$ sudo yum -y install trivy
+sudo vim /etc/yum.repos.d/trivy.repo
+ [trivy]
+ name=Trivy repository
+ baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$releasever/$basearch/
+ gpgcheck=0
+ enabled=1
+sudo yum -y update
+sudo yum -y install trivy
```

Comment:
These aren't copyable anyway, so I want to keep `$` to make it easier to understand which lines are commands.

Reply:
True, fair call this section does need manual intervention. Let me revert this one.